tests/utils.py

```diff
@@ -7,13 +7,61 @@
 import threading
 import subprocess
 import contextlib
-from tempfile import mkstemp
+from tempfile import mkstemp, mkdtemp, NamedTemporaryFile
+from loky.backend import resource_tracker
+from loky.backend.semlock import SemLock, _sem_open
+
+try:
+    FileNotFoundError = FileNotFoundError
+except NameError:  # FileNotFoundError is Python 3-only
+    from loky.backend.semlock import FileNotFoundError
```

Comment:
Nit: this could be simplified by always importing:

```python
from loky.backend.semlock import FileNotFoundError
```

Reply:
I don't think so, because `FileNotFoundError` is not a `global` of `loky.backend.semlock` for Python 3, it is a item of the `__builtins__`.